openpoke/decidim
# Implement sticky buttons for main CTAs in Mobile UI

#### :tophat: What? Why?
Implement sticky button for the main Call to Actions (CTAs) that remain accessible on the screen as the user scrolls through the detailed view of the Meetings, Proposals, Surveys, and Budgets components.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #12891
- Fixes #?

#### Testing
Visit PWA mobile browser
1. Meeting
  - Go to meetings.
  - Click on one of the meetings to see the sticky button to register if meeting registratión is available.
2. Proposals
  - In the menu, click on the button for the space where you want to see Proposals. For instance, it could be "Processes", 
    "Assemblies" or "Conferences".
  - Click on one of the highlighted options.
  - Click on the Jump to dropdown, and go to Proposals.
3. Survey
  - In the menu, click on the button for the space where you want to see the Survey. For instance, it could be "Processes", 
    "Assemblies" or "Conferences".
  - Click on one of the highlighted options.
  - Click on the Jump to dropdown, and go to Survey.
4. Budgets
  - In the menu, click on the button for the space where you want to see Budgets. For instance, it could be "Processes", 
    "Assemblies" or "Conferences".
  - Click on one of the highlighted options.
  - Click on the Jump to dropdown, and go to Budgets.
  
### :camera: Screenshots
Meeting 
<p align="center"><img width="200" alt="Meeting" src="https://github.com/openpoke/decidim/assets/116598037/c7ea876c-b209-4232-a2d3-e457adab0bc7"></p>

Poposals
<p align="center"><img width="200" alt="Proposal" src="https://github.com/openpoke/decidim/assets/116598037/4da3e55a-99c0-4b67-b9c9-54521f7792ee"><img width="200" alt="Proposals with visible support" src="https://github.com/openpoke/decidim/assets/116598037/ef836fba-bb62-4081-a56a-d60a65c88630"><img width="200" alt="Proposal with a threshold" src="https://github.com/openpoke/decidim/assets/116598037/32de1bcc-6c95-4abe-84ca-80fd2220d594"></p>

Survey
<p align="center"><img width="200" alt="Survey with no sections" src="https://github.com/openpoke/decidim/assets/116598037/f5deb2a6-9b09-49ba-b8fe-281981bef8d2"><img width="200" alt="Survey with sections (1/2)" src="https://github.com/openpoke/decidim/assets/116598037/9f20cda8-2f57-42b1-9f63-34694b258b72"><img width="200" alt="Survey with sections (2/2)" src="https://github.com/openpoke/decidim/assets/116598037/5bbaa7c4-29b0-4859-b53c-adfac9dcc0b9"></p>

Participatory Budget 
<p align="center"><img width="200" alt="Participatory Budget (1/4)" src="https://github.com/openpoke/decidim/assets/116598037/83eb1765-96ec-4dbc-b38a-bd683e815433"><img width="200" alt="Participatory Budget (2/4)" src="https://github.com/openpoke/decidim/assets/116598037/6d79d8ca-0ae7-473a-b9b4-506bdb7a1d43"><img width="200" alt="Participatory Budget (3/4))" src="https://github.com/openpoke/decidim/assets/116598037/db50bfbf-e1f9-4246-abea-081c9a1c0247"><img width="200" alt="Participatory Budget (4/4)" src="https://github.com/openpoke/decidim/assets/116598037/fe164b06-9143-4f60-9c3d-9dc1713b213c"></p>

:hearts: Thank you!
